app/controllers/verification_code_controller.rb

```diff
@@ -0,0 +1,65 @@
+class VerificationCodeController < BaseController
+  before_action :is_intake_locked
+  before_action :setup_edit
```

Comment:
set up edit happens before every action, so maybe this is just not the right name for this method